inpars/inpars.py

```diff
@@ -1,3 +1,296 @@
+# import os
```

Comment:
can we delete this commented block?

Reply:
That commented block is the updated inpars file generated by Danilo (with caching). I now use the regular InPars file for reproduction. We could remove It, but maybe save that one in a different file then